File: week5/simpleinheritance.java
class college {
    String collegeName = "MVGR";
    int collegeCode = 33;
}

class student extends college {
    String name = "Vamsi";
    int roll = 561;
}

class SimpleInherit {
    public static void main(String[] args) {
        student student1 = new student();
        System.out.println("Student name: "+student1.name);
        System.out.println("Student roll num : "+student1.roll);
        System.out.println("Student college name : "+student1.collegeName);
    }
}

File: week5/simpleinheritance.cpp
#include <iostream>
using namespace std;
class college
{
public:
    string collegeName = "MVGR";
    int collegeCode = 33;
};

class student : public college
{
public:
    string name = "Vamsi";
    int roll = 561;
};

int main()
{
    student obj;
    cout << "Student name : "<<obj.name << endl;
    cout << "Student roll num: "<<obj.roll << endl;
    cout << "College name: "<<obj.collegeName << endl;
    cout << "College code : "<<obj.collegeCode << endl;
